h2o-algos/src/main/java/hex/schemas/TreeV3.java

```diff
@@ -42,4 +42,9 @@
     @API(direction = API.Direction.OUTPUT, help = "Prediction values on terminal nodes")
     public float[] predictions;
 
+    @API(direction = API.Direction.OUTPUT, help = "Plain language rules representation of a trained decision tree")
+    public String language_tree_representation;
```

Comment:
Should the former `description` be removed and replaced by this ? Or `description` kept and the content change to this ? What's the point of previous `description` at the moment ? I'd get rid of it in favor of this.

Reply:
descriptions deprecated